errors.md

```
@@ -8,174 +11,317 @@ Errors consist of two parts: an error code and a message. Codes are universal,
 
-#### -1120 BAD_INTERVAL
+### -1120 BAD_INTERVAL
  * Invalid interval.
 
-#### -1121 BAD_SYMBOL
+### -1121 BAD_SYMBOL
  * Invalid symbol.
 
-#### -1125 INVALID_LISTEN_KEY
+### -1122 INVALID_SYMBOLSTATUS
+  * Invalid symbolStatus.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)